### PR TITLE
fix : Update FOSSASIA11 sample with new date format

### PR DESCRIPTION
**Pull request by Shailesh351**
Fixes part of #153 

Sample : FOSSASIA11

Changes : 
- Updated date format
- removed comma from some file which were causing issues in parsing

### CODE REVIEW

**Comment by Shailesh351** [comment]
@arp95 @iamareebjamal @mahikaw @shreyasnbhat @rohanagarwal94 Please review.

File: sample/FOSSASIA11/sessions [hunk 1]
```diff
@@ -5,8 +5,8 @@
         "subtitle": null,
         "short_abstract": "Short Abstract for the event",
         "long_abstract": "Welcome to the 3rd year of FOSSASIA. A friendly community of Open Technologists meets at the Science Centre Singapore.",
-        "start_time": "2011-11-11T08:30:00",
-        "end_time": "2011-11-1109:00:00",
+        "start_time": "2011-11-11T08:30:00+07:00",
+        "end_time": "2011-11-11T09:00:00+07:00",
```
Comment (iamareebjamal) [inline sample/FOSSASIA11/sessions]: Great work!

File: sample/FOSSASIA11/tracks [hunk 2]
```diff
@@ -12,19 +12,7 @@
             },
             {
                 "id": 2,
-                "title": "Mobile Solutions"
```
Comment (iamareebjamal) [inline sample/FOSSASIA11/tracks]: why removing tracks?

Comment (Shailesh351) [inline sample/FOSSASIA11/tracks]: @iamareebjamal Actually it is not tracks i have removed. It is sessions for this tracks. That don't exists in sessions file and was giving error while testing.

And this sample is not complete according to http://2011.fossasia.org

File: sample/FOSSASIA11/event [hunk 3]
```diff
@@ -3,10 +3,10 @@
...
     "latitude": 10.7627114,
     "longitude": 106.6911399,
-    "location_name": "Van Lang University Campus 2, 233A Phan Van Tri Street, Binh Thanh DistrictHo Chi Minh City",
-    "start_time": "2011-11-11T09:00:00",
-    "end_time": "2011-11-13T15:30:00",
-    "timezone": "UTC+07:00",
+    "location_name": "Van Lang University Campus 2, 233A Phan Van Tri Street, Binh Thanh District, Ho Chi Minh City",
+    "start_time": "2011-11-11T09:00:00+07:00",
+    "end_time": "2011-11-13T15:30:00+07:00",
+    "timezone": "Asia/Vietnum",
```
Comment (iamareebjamal) [inline sample/FOSSASIA11/event]: Please change to `Vietnam`

Comment (Shailesh351) [inline sample/FOSSASIA11/event]: @iamareebjamal I'm changing it to `Asia/Ho_Chi_Minh`
http://www.world-timedate.com/timezone/timezone_info_by_county.php?country_id=65

**Pull request merged**